knowledge-base/adr/0020-generator-options.md

```diff
@@ -43,7 +43,7 @@ We should discuss the details of those defaults and make a conscious decision on
 | `--outputDir`, `-o` _(required)_                         | `--outputDir`, `-o` _(required)_ | Output directory                                                                                            | _same_                                                                                                                                                                                                         |
 | `--clearOutputDir`                                       | `--clearOutputDir`               | Delete all files in output directory                                                                        | _same_                                                                                                                                                                                                         |
 | `--generateJs`                                           | `--transpile`, `-t`              | Transpiles, default true                                                                                    | Transpiles, default false. If set if tsconfig is enabled with default (unless configured). This should be explicitly stated in the documentation.                                                              |
-| `--generatePackageJson`                                  | `--[no]-packageJson`             | Writes a default package.json, default true.                                                                | Writes a default package.json, default true (needs core dependency). Optionally in the future: Writes a custom package.json if passed. Keep boolean for now.                                                   |
+| `--generatePackageJson`                                  | `--[no]-packageJson`             | Writes a default package.json, default true.                                                                | Writes a default package.json, default false (needs core dependency). Optionally in the future: Writes a custom package.json if passed. Keep boolean for now.                                                   |
```

Comment:
[req] This says "core dependency" which is 1. quite cryptic (what is meant a dependency to the @sap-cloud-sdk/core package) and 2. totally outdated as there is no such package anymore. Could you correct this?